src/tiles/list/tile-list.ts

```diff
@@ -305,7 +305,7 @@ export class TileList extends LitElement {
     if (!text) return nothing;
     return html`
       <div id=${ifDefined(id)} class="metadata">
-        ${this.labelTemplate(label)} ${text}
+        <p class="inline-wrap">${this.labelTemplate(label)} ${text}</p>
```

Comment:
Do we need the extra element? I found adding `max-height` to the `<div>` did the trick as well. A little less nesting :)

Reply:
These elements were originally added because Safari was entirely ignoring the line-clamp rule when the child wasn't a single inline element. But yeah, not necessary anymore with the `max-height` clamping. 🙂